spec/features/admin_spec.rb

```diff
@@ -36,5 +36,20 @@
     expect(page).to have_content("Leo")
   end
 
-
+  it "can add a new series" do
+    visit admin_index_path
+    page.click_link('Add a new series')
+    expect(current_path).to eq(new_admin_series_path)
+
+    VCR.use_cassette("admin_series") do
+      page.fill_in('series_name', with: "Cheers")
+      page.fill_in('series_seasons', with: "11")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.